source/_includes/post/date.html

```diff
@@ -7,9 +7,9 @@
 {% capture was_updated %}{{ updated | size }}{% endcapture %}
 
 {% if has_date != '0' %}
-  {% capture time %}<time datetime="{{ date | datetime | date_to_xmlschema }}" pubdate{% if updated %} data-updated="true"{% endif %}>{{ date_formatted }}</time>{% endcapture %}
```

Comment:
Since `date_formatted` is not used anymore, should we remove the line number 2?

`{% capture date_formatted %}{{ page.date_formatted }}{{ post.date_formatted }}{% endcapture %}`